### ISSUE TITLE
(Android) Pen pressure causes cursor to dissappear

### ISSUE THREAD

**Issue by ShadowFox1019** [ppy/osu#10228]
**Describe the bug:** 
When playing on the android version with a tablet and keyboard plugged in, the cursor disappears whenever pen pressure is applied. This makes it nearly unplayable when using a tablet. The only way to see the cursor is to hover. 
This bug first appeared when I updated osu!lazer to 2020.910.0. I was hoping it would be fixed before I made a bug report, however, it was not. 
Phone specifications:
Samsung galaxy s10
Android version 10
**Screenshots or videos showing encountered issue:** 
https://linksharing.samsungcloud.com/iB5HTAMTUOf4
**osu!lazer version:** 
2020.923.0
**Logs:**
https://linksharing.samsungcloud.com/kqeXpj1tUkkb
**Important note: link sharing is the only way I can figure out to attach these files on android. They have a time limit. If the time limit runs out it will not work. Please let me know if they run out, but I'll try to update them frequently so this does not happen.**

**Comment by smoogipoo** [comment]
This is arguably intended as you do have something on your screen to show your cursor - your pen. It's a different scenario from a mouse or graphics tablet or hovering your pen, where that's not the case.

Maybe the correct solution here is to have an option for this behaviour.

**Comment by peppy** [comment]
Can you confirm whether you're using an external tablet or whether your device is a touch device with pen input? This changes how we handle this issue.

**Comment by ShadowFox1019** [comment]
The device should be an external device as far as I know. If it matters at all, it's a gaomon s620 graphics tablet.

**Comment by mtoohey31** [comment]
Hi, I am also experiencing what I think is the same issue. I play on an iPad with an external monitor and peripherals. In the past I have played directly on the iPad's screen using an apple pencil with an external keyboard and screen tapping unmapped from clicking so I can use the pencil for only moving my cursor, the same way other people play with tablets. When I play this way, the cursor is dim which is fine cause I can see where the pencil's point is. 
However, I was attempting to play by looking at an external monitor that I had connected to the iPad, and when I do this, the cursor still dims, so I cannot tell where it is when I look at the external display instead of the iPad's screen. While my use case may not be very practical, there are others that would likely find the option to disable cursor dimming when on touchscreen (such as those trying to record/stream their gameplay from iOS or Android), to be a useful feature.
I would definitely appreciate it if this feature could be implemented at some point!
